src/main/java/org/folio/rest/impl/LocationAPI.java

```diff
@@ -177,7 +176,7 @@ public void getLocationsById(
     Context vertxContext) {
 
     String tenantId = getTenant(okapiHeaders);
-    Criterion criterion = idCriterion(id, LOCATION_SCHEMA_PATH, asyncResultHandler);
+    Criterion criterion = idCriterion(id, asyncResultHandler);
```

Comment:
What is the impact of not using the schema, does it change any validation that is done?